paymentapi/src/main/java/fi/hel/verkkokauppa/payment/api/OnlinePaymentController.java

```diff
@@ -2,10 +2,7 @@
 
 import fi.hel.verkkokauppa.common.error.CommonApiException;
 import fi.hel.verkkokauppa.common.error.Error;
-import fi.hel.verkkokauppa.payment.api.data.GetPaymentMethodListRequest;
-import fi.hel.verkkokauppa.payment.api.data.GetPaymentRequestDataDto;
-import fi.hel.verkkokauppa.payment.api.data.PaymentMethodDto;
-import fi.hel.verkkokauppa.payment.api.data.PaymentReturnDto;
+import fi.hel.verkkokauppa.payment.api.data.*;
```

Comment:
In general should avoid star imports (could be caused by IDE here). You wont know if there are 1 or 100 imports and what other dependencies the other classes cause.